MSTC-DA-IICT/Hacktoberfest23-ReACTivate
# Programming club

This pull request adds the Programming Club page to the application. The new page includes information about upcoming events organized by the Programming Club, as well as details about the club's mission, activities, and organizers.

Changes:
1. Created a new component for the Programming Club page.
2. Added countdown timers for each event, updating in real-time.

How to Test:
1. Navigate to the Programming Club page from the main menu.
2. Ensure that event details, club information, and organizer details are correctly displayed.
3. Verify that the countdown timers for upcoming events update in real-time.
4. Check that the page layout and images are visually appealing and functional.


@ManojDhundhalva Feel free to modify this template to suit your specific needs and the conventions of your development team or project.
![PrgClub](https://github.com/MSTC-DA-IICT/Hacktoberfest23-ReACTivate/assets/119687173/bd19c2b9-8b59-4e20-8d5d-0909ed32fdbe)
